draft-ietf-emu-eap-fido.md

```diff
@@ -334,121 +346,137 @@ After the TLS handshake is completed, the client and server perform the FIDO-exc
 
 This section describes the message format and the protocol flow.
 
-
 ### Message format
 
-All EAP-FIDO messages in the inner authentication consist of a CBOR sequence with one or two elements.
-Each message of the inner authentication MUST be sent in a single TLS record.
-The length of the TLS record MUST match the length of the CBOR sequence.
+All EAP-FIDO messages consist of a integer message type, optionally concatenated with a map of attributes.
+All elements are CBOR encoded. Note: this is not a CBOR sequence, but rather the binary concetenation of a CBOR
+integer and a CBOR map. This is illustrated in {{msgencoding}}
```

Comment:
This is actually a CBOR sequence, since CBOR sequence is defined as the concatenation of CBOR data items (see RFC 8742)

Reply:
It appears, that I conflated CBOR lists and CBOR sequences. Oops!

I guess here is an idea place to put a reference to what a CBOR sequence is then.

Reply:
Yep, I've already added it in my local working document, I'll push an update to the document soon.